packages/typescript/src/lib/TypescriptCompiler.ts

```diff
@@ -166,7 +166,75 @@ export class TypescriptCompiler {
     }
 
     // eslint-disable-next-line @typescript-eslint/no-var-requires
-    return require(tsconfigPath).compilerOptions;
+    const { compilerOptions } = require(tsconfigPath);
+
+    return this.normalizeTSConfigCompilerOptions(compilerOptions);
+  }
+
+  private normalizeTSConfigCompilerOptions(
+    options: ts.CompilerOptions
+  ): ts.CompilerOptions {
+    if (options.moduleResolution) {
+      const moduleResolution = (
+        options.moduleResolution as unknown as string
+      ).toLowerCase();
+      switch (moduleResolution) {
```

Comment:
I am looking at the typescript code.
Instead of this, I think that you can use the `getEmitModuleResolutionKind` function.
So, the result could be:
```
options.moduleResolution = ts.getEmitModuleResolutionKind(options)
```